subgraphs/uniswap-forks/protocols/vvs-finance/config/networks/cronos/cronos.ts

```diff
@@ -64,7 +64,7 @@ export class VSSFinanceCronosConfigurations implements Configurations {
     return FeeSwitch.ON;
   }
   getRewardIntervalType(): string {
-    return RewardIntervalType.NONE;
+    return RewardIntervalType.BLOCK;
```

Comment:
Same here

Reply:
It should have been BLOCK, but it does not need need to be re-deployed. There should be another option in the reward calculator. Currently, it does something like.... `If TIMESTAMP {handle like rewards are by timestamp};  else {handle like rewards are by block}. Therefore it will not change anything.

Reply:
SpritiswapV2 has been out for about a month now with a different set of contracts. So this change is irrelevant - though, we might want to consider indexing this one!